lib/pages/home_page.dart

```diff
@@ -0,0 +1,1437 @@
+import 'package:flutter/material.dart';
+
+import '../themes/colors.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      children: [
+        Column(
+          children: [
+            Container(
+              width: MediaQuery.of(context).size.width ,// 393,
+              height: MediaQuery.of(context).size.height, //852,
+              clipBehavior: Clip.antiAlias,
+              decoration: ShapeDecoration(
+                color:backgroundColor ,
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(24),
+                ),
+              ),
+              child: Stack(
+                children: [
+                  Positioned(
+                    left: 0,
+                    top: 59,
+                    child: Container(
+                      width: 393,
+                      height: 793,
+                      child: Column(
+                        mainAxisSize: MainAxisSize.min,
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Container(
+                            width: double.infinity,
+                            padding: const EdgeInsets.all(24),
+                            child: Row(
+                              mainAxisSize: MainAxisSize.min,
+                              mainAxisAlignment: MainAxisAlignment.start,
+                              crossAxisAlignment: CrossAxisAlignment.center,
+                              children: [
+                               const Expanded(
+                                  child: SizedBox(
+                                    child: Text(
+                                      'Expresso',
+                                      style: TextStyle(
+                                        color: Color(0xFF0F172A),
+                                        fontSize: 24,
+                                        fontStyle: FontStyle.italic,
+                                        fontFamily: 'Poppins',
+                                        fontWeight: FontWeight.w700,
+                                      ),
+                                    ),
+                                  ),
+                                ),
+                                const SizedBox(width: 24),
+                                Container(
+                                  padding: const EdgeInsets.symmetric(horizontal: 8, vertical: 4),
+                                  decoration: ShapeDecoration(
+                                    color: Color(0xFFFFE4E6),
```

Comment:
add const where all it's necessary. 

Use plugin/extension in code editor to flag such issues.